src/page/MoviePage.jsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import { connect } from 'react-redux';
+
 import { Paper } from '@material-ui/core';
 
 import { PAGE_COUNT } from '../constants/constants';
 import { MOVIES } from '../constants/actions';
-import Search from '../containers/Search';
-import MovieList from '../components/MovieList/MovieList';
+import Search from '../components/Search';
+import MovieList from '../components/MovieList';
 
-import { getMovies, getSearchMovies } from '../api/api';
+import { getMovies } from '../api/api';
 import { FlatPagination } from '../components/FlatPagination/FlatPagination';
 
+const mapStateToProps = ({ search, moviePage }) => ({
+  ...moviePage,
+  ...search,
+});
+
+const mapDispatchToProps = dispatch => ({
+  getActionDispatcher: action => () => dispatch(action),
+});
+
+@connect(mapStateToProps, mapDispatchToProps)
```

Comment:
Use anonymous functions instead `mapStateToProps` and `mapDispatchToProps`. Something like this 
`@connect(
  ({ search, moviePage }) => ({ search, moviePage }),
  { getMovies: getMoviesAction },
)`